README.md

```diff
@@ -1,3 +1,18 @@
+## Table of contents 
```

Comment:
It is nice to have `Table of contents` :+1: 

I think it would be more elegant to put it after [ActiveModelCachers](#activemodelcachers). What do you think?
There is also an repository doing so: https://github.com/ekalinin/github-markdown-toc#gh-md-toc

Reply:
Sounds good. I can remove the first item in the table of contents and move the rest of it to somewhere before "Compare with identity_cache".